packages/client/src/comp/Address/Container.ts

```diff
@@ -11,65 +11,45 @@ export const removeOptionalAddressFields = (fields: AddressInputParts) => ({
   unit: fields.unit ? fields.unit : undefined,
 })
 
-const handleDefaultState = (rawState: string) => {
-  // check if a state abbreviation is used
-  if (rawState.length === 2) {
-    const state = getState(rawState)
-    if (state) return state
-  }
-  if (isState(rawState)) {
-    return rawState
-  }
-
-  // Return an empty string to prevent errors with invalid states in the application
-  return ''
-}
-
 const handleDefault = (
   id: InputId,
   address: Address | null,
   path: Path | null,
   query: Record<string, string>,
 ): string => {
+  // Orgs' default value for every field has precedence over all others
+  // defaults, since they are always assigned as query params, this condition
+  // checks if they are present here.
+  if (query[id]) {
+    return query[id]
+  }
+
   // Path does not support 'postcode' but supports 'zip'
   const pathId = id === 'postcode' ? 'zip' : id
 
-  // We allow both the application and orgs to autofill the address input,
-  // orgs have the highest priority when we assign the default values.
   switch (path?.type) {
-    // The app itself can only auto-fill the state field when path === state.
-    // It's almost unlikely for the application to be loaded on this state,
-    // since this is the actual signup flow page; we coded this case here
-    // to avoid unexpected behavior.
-    case 'state':
-      if (pathId === 'state') {
-        return handleDefaultState(query[pathId] ?? path.state ?? address?.state ?? '')
-      }
-      return query[id] ?? ''
```

Comment:
Removed specific case since it's very unlikely for the application to load exactly at this state--but the default return for this switch statement still tries to assign a value from `address[id]`